### DISCUSSION TITLE
About Employment verification

### DISCUSSION THREAD

**Discussion by fgxtrue** [microsoft/MicrosoftEdge-Extensions#215] [Q&A]
Microsoft Partner Account Employment verification has been Rejected.But the information I provided is complete.
My account: aihome.chat@outlook.com

I have submitted feedback through the back-end and support center, but have not received a response.
Please review the materials I submitted and approve the Employment verification. Thank you
﻿
@microsoftopensource @sivsouvamMSFT @nagachaitanyalokam 
My domain: aihome.chat
My company: si chuan xing li ke ji you xian gong si（四川星力科技有限公司）
Domain whois: https://www.whois.com/whois/aihome.chat
[aihome.pdf](https://github.com/user-attachments/files/17537916/aihome.pdf)

**Comment by ManikanthMSFT** [comment]
Hi @fgxtrue, thank you for contacting us, we request you to please help us with your seller ID so that we can verify and assist you further.

**Reply by fgxtrue** [reply]
My Seller ID:87913040
Thank you for your reply.

**Reply by fgxtrue** [reply]
@ManikanthMSFT

**Reply by ManikanthMSFT** [reply]
Hi @fgxtrue, thank you for your response. We have forwarded your details to our team for further verification. We will update you as soon as we have more information.

To minimize confusion, could we please continue our conversation on a single query?

Thank you for your understanding.

**Reply by ManikanthMSFT** [reply]
Hi @fgxtrue, we got an update from our team stating they need the following Business and Domain verification documents to verify the account:

1) Acceptable document for business verification types include:
• Formation documents, such as articles of incorporation, partnership deed
• Franchise or agency appointment letters.
• Government issued letters, license, registration, or certificate.
• Lease or tenancy documents.
• Letter or statement from a financial institution or a utility company
• Record on a Government registry website (site/link must be displayed)
• Stock exchange filings or tax filing records.
• Certificate of incorporation or registration
• Extract from commercial register.
• Business license
• Tax certificate.
• DUNS certification

2) We need to verify that your company owns the domain that you used when you enrolled into Partner Center. This can be a Who Is lookup, or an invoice, purchase receipt for the domain, but it MUST meet the following criteria:

- List a domain that is an exact match to the one you are enrolling in Partner Center.
- Include your company's name (also match the company name you are enrolling into Partner Center)
- Have a purchase, renewal date not older than 12 months from the current date and not expiring in less than 60 days.

If it is not possible to provide additional documentation, please update your domain information to match the documents already provided.

**Reply by fgxtrue** [reply]
Hi @ManikanthMSFT. I have submitted the supporting documents mentioned by your team, but they have not been approved. Can you provide your email address so that I can send you all the relevant supporting documents.

**Reply by ManikanthMSFT** [reply]
Hey @fgxtrue, we have contacted our team regarding this. Once we receive an update, we will inform you promptly.

Thank you for your patience.

**Reply by fgxtrue** [reply]
![image](https://github.com/user-attachments/assets/c4e715c7-9527-4281-8724-721234bb71f2)
Thank thank you for your response @ManikanthMSFT . Now the Employment verification has passed, but the Business verification is Rejected. I don't know what happened and what to do next.

**Reply by ManikanthMSFT** [reply]
Hi @fgxtrue, could you please provide the acceptable document for business verification to our [email](ext_dev_support@microsoft.com):

• Formation documents, such as articles of incorporation, partnership deed
• Franchise or agency appointment letters.
• Government issued letters, license, registration, or certificate.
• Lease or tenancy documents.
• Letter or statement from a financial institution or a utility company
• Record on a Government registry website (site/link must be displayed)
• Stock exchange filings or tax filing records.
• Certificate of incorporation or registration
• Extract from commercial register.
• Business license
• Tax certificate.
• DUNS certification

**Reply by fgxtrue** [reply]
@ManikanthMSFT The Business license of my company has sended to the email:ext_dev_support@microsoft.com.
Please re-verify my account. Thank you.

**Reply by ManikanthMSFT** [reply]
Hi @fgxtrue, could you please help us with your email ID which you have sent the details for us to verify better.

**Reply by fgxtrue** [reply]
@ManikanthMSFT my email:aihome.chat@outlook.com

**Reply by ManikanthMSFT** [reply]
@fgxtrue, thank you for the confirmation. Can we continue our conversation through email to avoid confusion?

**Reply by fgxtrue** [reply]
@ManikanthMSFT Yes, we can

**Comment by ManikanthMSFT** [comment]
Closing this conversation.
